green_mbtools/mint/common_utils.py

```diff
@@ -359,7 +361,11 @@ def add_common_params(parser):
     parser.add_argument("--max_iter", type=int, default=100, help="Maximum number of iterations in the SCF loop")
     parser.add_argument("--keep_cderi", type=lambda x: (str(x).lower() in ['true','1', 'yes']), default='false', help="Keep generated cderi files.")
     parser.add_argument("--job", choices=["init", "sym_path", "ewald_corr"], default="init", nargs="+")
-    parser.add_argument("--x2c", type=lambda x: (str(x).lower() in ['true','1', 'yes']), default='false', help="enable X2C calculations")
+    parser.add_argument(
+        "--x2c", type=lambda x: (int(x) in [0, 1, 2]), default=0,
```

Comment:
I would suggest to use `choices` instead of using lambda, that would look clearer